Bottom: 904df4efcaa82ecdf00d70bf3dde1dafda6af2df
Top:    fb36e6669f63cca7c31fd10bade78e97c33f726b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-14 19:01:51 +0000

Refresh of improve-grid-snapping-heuristi

---

diff --git a/src/search.c b/src/search.c
index 85c8fe4..4d6c8db 100644
--- a/src/search.c
+++ b/src/search.c
@@ -498,9 +498,7 @@ SearchLinePointByLocation (int locked, LayerTypePtr * Layer,
   info.locked = (locked & LOCKED_TYPE) ? 0 : LOCKFLAG;
   if (r_search
       (SearchLayer->line_tree, &SearchBox, NULL, linepoint_callback, &info))
-  {
     return True;
-  }
   return False;
 }
 
@@ -1046,6 +1044,7 @@ IsPointInBox (LocationType X, LocationType Y, BoxTypePtr box, BDimension Radius)
 
 int ClosestArcPoint (float X, float Y, ArcTypePtr Arc)
 {
+  /* FIXME: Implement this function to return the distance to the arc */
   return 0;
 }
